docs/reference.rst

```diff
@@ -208,6 +208,26 @@ In this case you can't simply specify the same regexp for ``begin`` and
 ``end`` (say, ``"\\$[a-z]\\$"``), but you can use ``begin: "\\$[a-z]\\$"``
 and ``endSameAsBegin: true``.
 
+.. _endFilter:
+
+endFilter
```

Comment:
This was the original propose; do we still need it in presence of begin/end-plugins?

Reply:
Yes, it's on the chopping block.